code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -76,22 +120,37 @@
 	var/suits = list()
 	suits["amish suit"] = /obj/item/clothing/under/sl_suit
 	suits["black suit"] = /obj/item/clothing/under/suit_jacket
-	suits["blue suit"] = /obj/item/clothing/under/lawyer/blue
+	suits["black suit with skirt"] = /obj/item/clothing/under/suit_jacket/skirt
+	suits["black suit, alt"] = /obj/item/clothing/under/suit_jacket/alt
+	suits["black suit, alt skirt"] = /obj/item/clothing/under/suit_jacket/alt/skirt
+	suits["black lawyer suit"] = /obj/item/clothing/under/lawyer/black
+	suits["black lawyer skirt"] = /obj/item/clothing/under/lawyer/black/skirt
+	suits["blue lawyer suit"] = /obj/item/clothing/under/lawyer/blue
+	suits["blue lawyer skirt"] = /obj/item/clothing/under/lawyer/blue/skirt
 	suits["burgundy suit"] = /obj/item/clothing/under/suit_jacket/burgundy
 	suits["charcoal suit"] = /obj/item/clothing/under/suit_jacket/charcoal
 	suits["checkered suit"] = /obj/item/clothing/under/suit_jacket/checkered
 	suits["executive suit"] = /obj/item/clothing/under/suit_jacket/really_black
+	suits["executive skirt"] = /obj/item/clothing/under/suit_jacket/really_black/skirt
+	suits["gentleman suit"] = /obj/item/clothing/under/gentlesuit
```

Comment:
If this #8941's description is the main basis of argument here, I'm going to make my arguments targeting them:

With that, bless Wezzy, but I don't see how "a bunch of useless fucking shit that isn't even available in the loadout and hurts my eyes like what is this shit" (very much paraphrased) constitutes a very solid argument—especially when considering how one of them was actually on the loadout and a lot of this seems to be done based on personal preference rather than the choices of other players in the community. Granted, some of them were indeed leftover custom items, but I've deliberately avoided adding those back in because those are custom items and in my eyes (and I stress **MY** eyes) are almost like the de facto "property" of their owners.
However, I can tell you for myself in the humbly limited time I have been here that I know quite a few of these AREN'T custom items.

Likewise, I'm not entirely sure how this is specifically causing any issues.

Reply:
I, for one, used the black suit across multiple characters of mine. I recently noticed that it was gone and would be glad to see it come back.

Reply:
@alsoandanswer so this won't go behind you